dev-tools/packaging/packages.yml

```diff
@@ -301,6 +307,65 @@ shared:
         source: '{{ elastic_beats_dir }}/specs/connectors.spec.yml'
         mode: 0644
 
+  - &agent_docker_edot_spec
+    <<: *agent_binary_spec_no_components
+    docker_variant: 'edot-collector'
```

Comment:
The third name change is the charm: https://github.com/elastic/elastic-agent/issues/6764#issuecomment-2730506335

`elastic-otel-collector`

Reply:
I believe this has already been addressed and now the docker_variant for edot points to `elastic-otel-collector` and `elastic-otel-collector-wolfi`. Please unresolve if you spot anything else 🙂